common/build/eslint-config-fluid/minimal.js

```diff
@@ -121,7 +121,9 @@ module.exports = {
         "@typescript-eslint/member-ordering": "off",
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/no-parameter-properties": "off",
-        "@typescript-eslint/no-unused-vars": "off",
+        // Disallow unused variables except those named explicitly to be ignored.
+        // Allows patterns like `const [_, foo] = bar();`
+        "@typescript-eslint/no-unused-vars": ["error", { args: "none", varsIgnorePattern: "^_" }],
```

Comment:
Let's disallow everything.  At least currently an underscore prefix in our repo is usually used for a private member that may also have a public getter, or otherwise-shadowed variables -- neither of which should be unused.

Reply:
If something merits an exception, we can do it with an `// eslint-disable-next-line` instead which is easier to find later.

Reply:
I actually saw this pattern used pretty widely. I initially set out to disable everything, but we have quite a few places (on the order of ~100) instances of this convention being used.

Reply:
Are they contained to any particular package?  I'd rather disable at the package level if we have misbehaving packages rather than impact the rest of the repo

Reply:
That's fair. I'll double check, but I think they were pretty spread out.

Reply:
Also, fwiw the pattern you mention above can just be `const [, foo] = bar();` to avoid the unused.

Reply:
Oh, that's right. Forgot you could do that. Thanks!